lmctl/client/api/auth.py

```diff
@@ -42,6 +44,41 @@ def request_user_access(self, client_id: str, client_secret: str, username: str,
         auth_response = self.base_client.make_request_for_json(request)
         return auth_response
 
+    def request_okta_user_access(self, client_id: str, client_secret: str, username: str, password: str, scope: str = None, auth_server_id: str = None, okta_server: str = None) -> Dict:
+        auth = self._build_client_basic_auth(client_id, client_secret)
+        body = {
+            'username': username,
+            'password': password,
+            'grant_type': 'password',
+        }
+        if scope is not None:
+            body["scope"] = scope
+        request = TNCOClientRequest(method='POST', endpoint=self.okta_authserver_endpoint.format(auth_server_id))\
+                        .disable_auth_token()\
+                        .add_form_data(body)\
+                        .add_auth_handler(auth)
+        if okta_server:
+            request.override_address = okta_server
+        auth_response = self.base_client.make_request_for_json(request)
+        return auth_response
+
+    def request_okta_access(self, client_id: str, client_secret: str, scope: str = None, auth_server_id: str = None, okta_server: str = None) -> Dict:
```

Comment:
This should probably be `request_okta_client_access`?

Reply:
Addressed